ui/app/models/database/role.js

```diff
@@ -68,52 +79,63 @@ export default class RoleModel extends Model {
     helperTextEnabled: 'Vault will rotate password after.',
   })
   rotation_period;
-  @attr({
-    label: 'Skip initial rotation',
-    editType: 'toggleButton',
-    defaultValue: false, // this defaultValue will be set in database-role-setting-form.js based on parent database value
-    helperTextDisabled: 'Vault will rotate password for this static role on creation.',
-    helperTextEnabled: "Vault will not rotate this role's password on creation.",
-  })
-  skip_import_rotation;
+
   @attr('array', {
     editType: 'stringArray',
   })
   creation_statements;
+
   @attr('array', {
     editType: 'stringArray',
     defaultShown: 'Default',
   })
   revocation_statements;
+
   @attr('array', {
     editType: 'stringArray',
     defaultShown: 'Default',
   })
   rotation_statements;
+
   @attr('array', {
     editType: 'stringArray',
     defaultShown: 'Default',
   })
   rollback_statements;
+
   @attr('array', {
     editType: 'stringArray',
     defaultShown: 'Default',
   })
   renew_statements;
+
   @attr('string', {
     editType: 'json',
     allowReset: true,
     theme: 'hashi short',
     defaultShown: 'Default',
   })
   creation_statement;
+
   @attr('string', {
     editType: 'json',
     allowReset: true,
     theme: 'hashi short',
     defaultShown: 'Default',
   })
   revocation_statement;
+
+  // ENTERPRISE ONLY
+  @attr({
+    label: 'Rotate immediately',
+    editType: 'toggleButton',
+    defaultValue: true, // this defaultValue will be set in database-role-setting-form.js based on parent database value
```

Comment:
Thinking more about this, do we need to set a default here if it's always being set by the parent?